modules/jlink/artifacts/opt/jboss/container/java/s2i/README.md

```diff
@@ -0,0 +1,37 @@
+# jlink-openshift App builder
```

Comment:
I don't think we need this file

Reply:
New path for this file in current PR is modules/jlink/artifacts/opt/jboss/container/java/jlink/README.md -- I still think we don't need it